tool/tsh/proxy.go

```diff
@@ -355,13 +355,23 @@ func onProxyCommandDB(cf *CLIConf) error {
 		}
 	}()
 
+	onNewConnection := func(dbCert *x509.Certificate, lp *alpnproxy.LocalProxy, conn net.Conn) {
+		if time.Now().After(dbCert.NotAfter) {
+			fmt.Fprintln(cf.Stdout())
+			fmt.Fprintln(cf.Stdout(), "Your database session is expired. Please restart the local proxy.")
```

Comment:
Why should they restart the local proxy? Since the database cli is providing the certs and the local proxy is just forwarding, can't they just use `tsh db login` and it will work? This works without `--tunnel` already, they just need to know to do it

Reply:
This message is shown at the end of `tsh proxy db` command to ask the user to rerun whatever `tsh proxy db` command they are currently running.

Reply:
Oh I see - does `tsh proxy db` not work with reissued certs if it is started with expired certs? I actually haven't tried that